scripts/plots/analyses/exevs_analyses_grid2obs_plots.sh

```diff
@@ -165,6 +165,25 @@ do
 	then
 	echo "WARNING: NO PLOT FOR",$varb,$region,$anl
         fi
+
+	if [ ! -e $COMOUTplots/$varb/evs.${anl}.bcrmse_me.${smvar}_${smlev}.last31days.timeseries.buk_${smregion}.png ]
+	then
+	$PARMevs/metplus_config/${STEP}/${COMPONENT}/${VERIF_CASE}/py_plotting.config_timeseries
+	export err=$?; err_chk
+        else
+	echo "RESTART - plot exists; copying over to plot directory"
+	cp $COMOUTplots/$varb/evs.${anl}.bcrmse_me.${smvar}_${smlev}.last31days.timeseries.buk_${smregion}.png $PLOTDIR
+	fi 
+
+	if [ -e ${PLOTDIR}/sfc_upper/*/evs*png ]
+	then
+        mv ${PLOTDIR}/sfc_upper/*/evs*png ${PLOTDIR}/evs.${anl}.bcrmse_me.${smvar}_${smlev}.last31days.timeseries.buk_${smregion}.png
+	cp ${PLOTDIR}/evs.${anl}.bcrmse_me.${smvar}_${smlev}.last31days.timeseries.buk_${smregion}.png $COMOUTplots/$varb
```

Comment:
Add check for if SENDCOM=YES before copying to COMOUT.

This comment is for it happening here and other instances in this script.

Reply:
OK, will do.

Reply:
The file check is originally for `${PLOTDIR}/sfc_upper/*/evs*png` but the copy to COMOUT is for `${PLOTDIR}/evs.${anl}.bcrmse_me.${smvar}_${smlev}.last31days.timeseries.buk_${smregion}.png`

Reply:
In this example

Reply:
Yes, but it's basically the same file since it's mv'd in the previous statement.  I cannot see how the file will ever not exist if we are inside the if loop that we are in.  

I'll do it, but I think it's redundant.  (Especially when I do the future PR that will put the full name of the file in the original if statement).

Reply:
Ah yeah I forgot about that PR too. We can leave it then. I'll mark this as resolved and then get to testing.

Reply:
@malloryprow Additional file checks are now in.